modules/partials/header.php

```diff
@@ -1,57 +1,74 @@
 <?php
-// Define an array to contain page titles
 $pageTitles = array(
     'home' => "Home",
+    'login' => "Login",
     "profile" => "My Profile",
+    "register" => "Register", // Page title for Regiser
+    "change_password" => "Change Password"
 );
-// Get page title depend on what is using module. 
-$pageTitle = $pageTitles[$module];
+$pageTitle = isset($pageTitles[$module]) ? $pageTitles[$module] : null;
+
+$userId = isset($_SESSION["login_user_id"]) ? $_SESSION["login_user_id"] : null;
+
+$user = false;
+
+if ($userId) {
+    $sql = "SELECT id, username, email, fullname, password
+    FROM users
+    WHERE id = $userId
+    LIMIT 0,1";
+
+    $result = $mysql->query($sql);
+
+    $user = $result->fetch_array() ?? false;
+}
+
+$fullname = $user['fullname'] ? $user['fullname'] : 'Guest';
+
+$loginOrLogout = $user['fullname'] ? 'Logout' : 'Login';
+
 ?>
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <link rel="stylesheet" href="normalize.css">
-        <meta charset="utf-8">
-        <!-- use defined variable to render page title in HTML -->
-        <title><?php echo $pageTitle; ?></title>
-        <link rel="stylesheet" href="./assets/css/index.css" />
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-
-        <script src="./assets/js/index.js"></script>s
-    </head>
-    <body>
-        <!-- The Header -->
-        <header> 
-        <div>
-            <h4>The logo</h4>
-        </div>
-        <div>
-                <h2 class="slogan">The header slogan</h2>
+  <head>
+    <meta charset="utf-8">
+    <title><?php echo $pageTitle; ?></title>
+    <link rel="stylesheet", href="./assets/css/index.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="./assets/js/index.js"></script>
+  </head>
+  <body>
+    <header>
+        <div id="logo">
+            <h4>The logo<?php echo $userId; ?></h4>
         </div>
+        <div id="slogan">The header slogan</div>
         <div id="form">
             <ul>
-                <li>Hi <span>Guest</span></li>
-                <li><a href="javascript:void(0)" onclick="showLoginForm()">Login</a></li>
+                <li>Hi <span><?php echo $fullname; ?></span></li>
+                <li><a href=<?php if ($loginOrLogout === 'Login') {echo "javascript:void(0)";} else {echo "./index.php?m=logout";} ?> id="form1" onclick=<?php if ($loginOrLogout === 'Login'){echo "showLoginForm()";} ?>><?php echo $loginOrLogout ?></a></li>
```

Comment:
Make your code to readable by change the logic like:

```php
<?php 
if (!$user) { ?>
    <li><a href="javascript:void(0)" onclick="showLoginForm()">Login</a></li>
<?php } else { ?>
    <li><a href="index.php?m=logout">Logout</a></li>
<?php } ?>
```

Please aways try to keep your code to more simple.